dcc-submission-ui/dictionary/app/index.html

```diff
@@ -41,9 +42,19 @@
         <script src="bower_components/regex-colorizer/regex-colorizer.js"></script>
         <script src="bower_components/d3-tip/index.js"></script>
         <script src="bower_components/jsoneditor/dist/jsoneditor.js"></script>
+        <script src="bower_components/jsondiffpatch/public/build/jsondiffpatch-full.min.js"></script>
 
-        <script src="scripts/vendor/beautify.js"></script>
+        <script>
+            var jsondiffpatch = jsondiffpatch.create({
```

Comment:
Its required if we want to show changes in lines < 60 character in length, like the file name pattern.

https://github.com/benjamine/jsondiffpatch#options